super_table_pyspark.py

```diff
@@ -0,0 +1,547 @@
+from pprint import pprint
+import re
+
+
+from helper import (
+    get_req_handler,
+    getCurrentDateTime,
+    initBoto3Session,
+    get_s3_objs,
+    formatted_timestamp,
+)
+from csv_schemas import *
+from dotenv import load_dotenv
+import boto3
+import os
+from io import StringIO
+
+import pandas as pd
+import json
+
+from math import sin, cos, sqrt, atan2, radians
+
+from pyspark.sql import SparkSession
+from pyspark.sql.functions import *
+from pyspark.sql.types import *
+from pyspark.sql.window import Window
```

Comment:
Do add the dependencies installed from pip into requirements.txt
In the dir, run 
```
pip freeze > requirements.txt
```